src/lib.mo

```diff
@@ -105,30 +105,21 @@ module {
       initialFee,
       triggerOnNotifications,
       freezeTokenHandler,
-      func(p : Principal, x : Int) { creditRegistry.issue(#user p, x) },
+      creditRegistry,
```

Comment:
I would like to avoid passing creditRegistry to AccountManager. Original idea was to have AccountManager and CreditRegistry two independent modules used by TokenHandler. Now CreditRegistry becomes a submodule of AccountManager.